### PR TITLE
Cleanup JSON_INFO.md

### PR DESCRIPTION
**Pull request by Cyrano7**
Just general cleanup of this doc as in localization, spelling, grammar, syntax. Had most of these around awhile and want to update more but might as well get what I have in before 0.F 

#### Summary
None

### CODE REVIEW

File: doc/JSON_INFO.md [hunk 1]
```diff
@@ -903,11 +904,11 @@ If a fuel has the PERPETUAL flag, engines powered by it never use any fuel.  Thi
...
 | `is_safe`   | (bool) Check to not trigger safe-mode warning, currently inconsequential.
 | `is_animal` | (bool) Check if that group has only normal animals, currently inconsequential.
-| `freq_total`| (int) Determines the number of entries created for the monster roll, default 1000. If the total eligable `freq`s of a group exceed `freq_total` the entries after the monster that exceeded it **won't be included** in the roll - ie if the first two monsters out of a group of ten each have `freq: 500` the rest of the group won't have a chance to spawn at all!
-| `replace_monster_group` | (bool) Check if the group should be replaced completely by another monster group as game time progresses - doesn't affect already spawned monsters, as such mostly superceded by monster evolution.
+| `freq_total`| (int) Determines the number of entries created for the monster roll, default 1000. If the total eligible `freq`s of a group exceed `freq_total` the entries after the monster that exceeded it **won't be included** in the roll - ie if the first two monsters out of a group of ten each have `freq: 500` the rest of the group won't have a chance to spawn at all!
```
Comment (actual-nh) [inline doc/JSON_INFO.md:910]:
```suggestion
| `freq_total`| (int) Determines the number of entries created for the monster roll, default 1000. If the total eligible `freq`s of a group exceed `freq_total` the entries after the monster that exceeded it **won't be included** in the roll - i.e., if the first two monsters out of a group of ten each have `freq: 500` the rest of the group won't have a chance to spawn at all!
```

File: doc/JSON_INFO.md [hunk 2]
```diff
@@ -2446,8 +2447,8 @@ Pet armor can be defined like this:
 "environmental_protection" : 0,  //  (Optional, default = 0) How much environmental protection it affords
 "material_thickness" : 1,  // Thickness of material, in millimeter units (approximately).  Generally ranges between 1 - 5, more unusual armor types go up to 10 or more
 "pet_bodytype":        // the body type of the pet that this monster will fit.  See MONSTERS.md
-"max_pet_vol:          // the maximum volume of the pet that will fit into this armor. Volume in ml and L can be used - "50 ml" or "2 L".
-"min_pet_vol:          // the minimum volume of the pet that will fit into this armor. Volume in ml and L can be used - "50 ml" or "2 L".
+"max_pet_vol":          // the maximum volume of the pet that will fit into this armor. Volume in ml and L can be used - "50 ml" or "2 L".
+"min_pet_vol":          // the minimum volume of the pet that will fit into this armor. Volume in ml and L can be used - "50 ml" or "2 L".
```
Comment (actual-nh) [inline doc/JSON_INFO.md:2451]:
```suggestion
"max_pet_vol":          // the maximum volume of the pet that will fit into this armor. Volume in ml or L can be used - "50 ml" or "2 L".
"min_pet_vol":          // the minimum volume of the pet that will fit into this armor. Volume in ml or L can be used - "50 ml" or "2 L".
```

File: doc/JSON_INFO.md [hunk 3]
```diff
@@ -2783,7 +2784,7 @@ Alternately, every item (book, tool, armor, even food) can be used as a gunmod i
 "charges_per_use": 1,      // Charges consumed per tool use
 "initial_charges": 75,     // Charges when spawned
 "max_charges": 75,         // Maximum charges tool can hold
-"rand_charges: [10, 15, 25], // Randomize the charges when spawned. This example has a 50% chance of rng(10, 15) charges and a 50% chance of rng(15, 25) (The endpoints are included)
+"rand_charges": [10, 15, 25], // Randomize the charges when spawned. This example has a 50% chance of rng(10, 15) charges and a 50% chance of rng(15, 25) (The endpoints are included)
```
Comment (actual-nh) [inline doc/JSON_INFO.md:2787]:
```suggestion
"rand_charges": [10, 15, 25], // Randomize the charges when spawned. This example has a 50% chance of rng(10, 15) charges and a 50% chance of rng(15, 25). (The endpoints are included.)
```

File: doc/JSON_INFO.md [hunk 4]
```diff
@@ -2949,7 +2950,7 @@ The contents of use_action fields can either be a string indicating a built-in f
     "need_worn": true;                        // Whether the item must be worn to be transformed; false by default.
     "need_wielding": true;                    // Whether the item must be wielded to be transformed; false by default.
     "target_charges" : 3, // Number of charges the transformed item has.
-    "rand_target_charges: [10, 15, 25], // Randomize the charges the transformed item has. This example has a 50% chance of rng(10, 15) charges and a 50% chance of rng(15, 25) (The endpoints are included)
+    "rand_target_charges": [10, 15, 25], // Randomize the charges the transformed item has. This example has a 50% chance of rng(10, 15) charges and a 50% chance of rng(15, 25) (The endpoints are included)
```
Comment (actual-nh) [inline doc/JSON_INFO.md:2953]:
```suggestion
    "rand_target_charges": [10, 15, 25], // Randomize the charges the transformed item has. This example has a 50% chance of rng(10, 15) charges and a 50% chance of rng(15, 25). (The endpoints are included.)
```

File: doc/JSON_INFO.md [hunk 5]
```diff
@@ -3298,9 +3299,9 @@ Array of dictionaries defining possible items produced on butchering and their l
     `bionic`: an item gained by dissecting the creature. not restricted to CBMs.
     `bionic_group`: an item group that will give an item by dissecting a creature. not restricted to groups containing CBMs.
 
-`flags` value should be an array of strings.  It's the flags that will be added to te items of that entry upon harvesting.
+`flags` value should be an array of strings.  It's the flags that will be added to the items of that entry upon harvesting.
```
Comment (actual-nh) [inline doc/JSON_INFO.md:3302]:
```suggestion
`flags` value should be an array of strings.  These flags will be added to the items of that entry upon harvesting.
```

File: doc/JSON_INFO.md [hunk 6]
```diff
@@ -3298,9 +3299,9 @@ Array of dictionaries defining possible items produced on butchering and their l
...
 
-`faults` value should be an array of `fault_id` strings.  It's the faults that will be added to te items of that entry upon harvesting.
+`faults` value should be an array of `fault_id` strings.  It's the faults that will be added to the items of that entry upon harvesting.
```
Comment (actual-nh) [inline doc/JSON_INFO.md:3304]:
```suggestion
`faults` value should be an array of `fault_id` strings.  These faults will be added to the items of that entry upon harvesting.
```

File: doc/JSON_INFO.md [hunk 7]
```diff
@@ -3309,7 +3310,7 @@ For every `type` other then `bionic` and `bionic_group` following entries scale
     `mass_ratio` value is a multiplier of how much of the monster's weight comprises the associated item. to conserve mass, keep between 0 and 1 combined with all drops. This overrides `base_num`, `scale_num` and `max`
 
 
-For `type`s: `bionic` and `bionic_group` following enrties can scale the results:
+For `type`s: `bionic` and `bionic_group` following entries can scale the results:
```
Comment (actual-nh) [inline doc/JSON_INFO.md:3313]:
```suggestion
For `type`s `bionic` and `bionic_group`, the following entries can scale the results:
```

**Review by actual-nh — COMMENTED** [review]
Good spotting!

**Comment by int-ua** [comment]
Please fix the conflict

**Comment by Cyrano7** [comment]
@int-ua Should be all good now.

**Review by int-ua — APPROVED** [review]

**Pull request merged**